src/pages/Software.pug

```diff
@@ -92,7 +92,116 @@ block article
+        Figure 3 shows an example of such a file.
+        Each read from sequencing is expressed by four lines.
+        While the first and third line can contain an identifier and description of the sequence, the second and fourth line are more important to us.
+        The second line contains the actual DNA sequence.
+        In line four the quality of the sequence is encoded.
+        This is important to assess how confident the sequencing was.
+        The quality is encoded in values reaching from 0x21 (33 in decimal) to 0x7E (126) and displayed as the corresponding ASCII character.
+        So a "!" marks the lowest quality, while a "~" stands for the highest quality.
+
+        ## Removing Primers and Cleaning Up Data
+
+    :markdown-it
+
+        The first step necessary for decoding is the removal of the primer since this does not contain any relevant information.
+        To do so, the software scans every sequence for a subsequence that is similar to the primer.
+        If it finds the primer in a sequence, it deletes the primer and everything prior to it from the sequence.
+        If it does not find a primer, it deletes the whole sequence from the dataset.
+        As shown in figure 4, the sequence encoding information begins right behind the primer.
+        So by removing the primer and everything before it, we know exactly that the sequence now begins with the encoding sequence.
+        After that, this step is repeated with the reverse primer.
+        The reverse primer starts directly behind the encoding sequence.
+        After removing it and everything behind it, all sequences now only contain the relevant information.
+
+    - var imgpath = "" + require("../assets/img/Software/sequence_overview.svg")["default"]
+    +image(4, imgpath, "Overview of the components of a sequence.", 90, 1, true)
+
+    :markdown-it
+
+        Before the primer, there is an additional sequence added.
+        This is necessary for Nanopore Sequencing.
+        By clearing the sequences in the way described before, these are also removed automatically.
+        If there is any sequence not containing one of the primers, it will be removed completely.
+        This ensures that no corrupted data disturb the further steps.
+
+        ### Finding Primer Sequences with Errors
+
+        When searching for primers in the sequences, not only perfect matches should be found.
+        In synthesis and sequencing, errors can happen in various places.
+        So all subsequences that are *very similar* to a primer should be classified as such.
+        For that classification, the levenshtein distance is used.
+        This is a very popular measure for all kinds of string comparisons.
+        It describes how many insertions, deletions, or substitutions of symbols are needed to match two strings.
+        For example, the seqeunces "ACGT" and "ACGA" have a levenshtein distance of 1, since one substitution is needed to match them.
```

Comment:
```suggestion
        For example, the sequences "ACGT" and "ACGA" have a Levenshtein distance of 1, since one substitution is needed to match them.
```